apps/app/src/Screens/Friends.tsx

```diff
@@ -0,0 +1,17 @@
+import { Menu } from "@/components/Menu/Menu"
+import { View, Text } from "react-native"
+
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { RootStackParamList } from "./Types/Stack";
+
+type FriendsProps = NativeStackScreenProps<RootStackParamList, "Friends">
+
+export const Friends = ({navigation, route}: FriendsProps) => {
```

Comment:
**suggestion:** Since you don't export this type and it's just used to type the props of this specific component, it's not really necessary to specify it in the name here. It's definitely not wrong and very minor, but might same you some time when typing or renaming components

```suggestion
type Props = NativeStackScreenProps<RootStackParamList, "Friends">

export const Friends = ({navigation, route}: Props) => {
```

Reply:
Cool
[renaming props](https://github.com/dbrekelmans/questline-galaxy/pull/4/commits/ea89ae7815980e4395b89782ca08a74ff277fd1e)